devtools/conda-envs/test_env.yaml

```diff
@@ -32,7 +32,11 @@ dependencies:
   - qcelemental=0.25.1
   - qcportal>=0.50
   - requests
+  - jax
+  - flax
```

Comment:
can we avoid `jax` and `flax` in this implementation?

Reply:
I was implementing correctness tests against the original, should we remove this from the testing suite?